src/Sarif.Driver/DriverExtensionMethods.cs

```diff
@@ -12,12 +12,13 @@ public static class DriverExtensionMethods
     {
         public static LoggingOptions ConvertToLoggingOptions(this AnalyzeOptionsBase analyzeOptions)
         {
-            LoggingOptions loggingOptions = LoggingOptions.None;
+            LoggingOptions loggingOptions = LoggingOptions.PrettyPrint;
```

Comment:
>PrettyPrint [](start = 59, length = 11)

Here's our behavior now, if neither minify or pretty print is specified, we pretty print. If only minify is specified, we minify, if both minify and pretty print are specified, we pretty print. We could consider raising an exception instead for the case where both settings exist.

Reply:
In this case, i think we are missing one validation in the ValidateOptions method.
If we check, we have another implementation `ValidateOutputFormatOptions` that checks exactly for this behavior

---
In reply to: [530638428](https://github.com/microsoft/sarif-sdk/pull/2168#discussion_r530638428) [](ancestors = 530638428)

Reply:
Taking a second look, we wouldn't be able to use just the way it is because it uses another class as option. In the future, we could refactor this.

---
In reply to: [530655298](https://github.com/microsoft/sarif-sdk/pull/2168#discussion_r530655298) [](ancestors = 530655298,530638428)

Reply:
You're right, we should make this a general helper in the driver.

---
In reply to: [530675864](https://github.com/microsoft/sarif-sdk/pull/2168#discussion_r530675864) [](ancestors = 530675864,530655298,530638428)